lib/clone.js

```diff
@@ -49,21 +52,24 @@ const cloneTask = (sshGitHubUrl, repoName, progressBar) => {
 const clone = (services, task = cloneTask) => {
   process.stdout.write(`Cloning repos...${newline}`)
 
-  const gitHubDetails = getGitHubDetails(services)
+  const gitHubRepoInfo = getGitHubDetails(services)
   const progressBar = new Progress(':percent [:bar] (:current/:total)', {
-    total: gitHubDetails.length,
+    total: gitHubRepoInfo.length,
     clear: true
   })
+  const githubInfoParentArray = chunkArray(gitHubRepoInfo, numberOfCPUs * 4)
```

Comment:
Agreed, what would you name the `const` to communicate this meaning?

`const ???? = 4`